platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -250,5 +250,26 @@
             'hot_swappable': True,
             'led_num': 1
         }
+    },
+    'x86_64-mlnx_msn4600-r0': {
+        'thermal': {
+            'minimum_table': {
+                "unk_trust": {"-127:40": 12, "41:120": 13},
```

Comment:
for these numbers please review with Vadim P
we can only approve them after platform test is fully running on this system.

Reply:
I talked with Vadim, he didn't have time for doing a full review but told me to check that it the same as the Leopard, I checked and they are only different on the fans and unk, for those change he said that fans are correct, and unk should be the same as Tigon which it is.